app/src/main/java/ch/epfl/sdp/peakar/user/services/AccountData.java

```diff
@@ -0,0 +1,80 @@
+package ch.epfl.sdp.peakar.user.services;
+
+import android.net.Uri;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import ch.epfl.sdp.peakar.points.CountryHighPoint;
+import ch.epfl.sdp.peakar.points.POIPoint;
+import ch.epfl.sdp.peakar.user.challenge.Challenge;
+import ch.epfl.sdp.peakar.user.friends.FriendItem;
+
+public class AccountData {
```

Comment:
Missing JavaDoc on everything in class. Both the class itself and the functions. 
However most are getters, not sure about our convetion regarding comments on getters.

Reply:
Other comment noticed when I looked in other classes. I see that we add peaks heights etc. By doing a get and then adding. I think it is better praxis to have an add function in this class. However I am not sure why it is better praxis. I have just heard it. What do you think?